hazelcast/src/main/java/com/hazelcast/config/WanPublisherConfig.java

```diff
@@ -48,10 +48,27 @@ public WanPublisherConfig setGroupName(String groupName) {
         return this;
     }
 
+    /**
+     * Get the capacity of the queue for WAN replication events. IMap, ICache, normal and backup events count against
+     * the queue capacity separately. When the queue capacity is reached, backup events are dropped while normal
+     * replication events behave as determined by the {@link #getQueueFullBehavior()}.
+     * The default queue size for replication queues is 10000.
```

Comment:
You could use `{@value #DEFAULT_QUEUE_CAPACITY}` here, so you don't have a copy&paste value of the source code in the JavaDoc.